### DISCUSSION TITLE
Support Data Links in Status History panel

### DISCUSSION THREAD

**Discussion by domclassen** [grafana/grafana#38318] [Dashboards, Panels & Visualizations]
Currently you can configure data links in Status History panels but they don't show or function in 8.x (tested with 8.0.3).
This should ideally work just the same as in 7.x where we were using the community predecessors of that panel.

The intention is to have an aggregated view on status and then be able to drill into details, carrying over field values into the target dashboard.

**Comment by dsotirakis** [comment]
Hello @domclassen, thanks for raising this. Please can you use the template for creating features next time you create one?  Thanks!
```
<!-- Please only use this template for submitting feature requests -->

**What would you like to be added**:

**Why is this needed**:
```

**Comment by VolkrB** [comment]
I would like for this to also be added - Data links working in Status History Panels

I think this should already work, is it a bug?

edit: Grafana Version 8.1.1

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by VolkrB** [comment]
Just to clarify, the option is there to set Data Links, but they don't work:
Is this something that was meant to be implemented in the future, an oversight or a bug?

![image](https://user-images.githubusercontent.com/20803912/130301193-0750b7b1-8245-4ad3-a4a5-e646f6035840.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
